incuna-sass/_normalize.sass

```diff
@@ -1,5 +1,5 @@
 // Incuna overrides
-@if not support-legacy-browser('ie', '7')
+.lt-ie8 &
```

Comment:
Need to explain how these classes are used, where they are placed, how they are made, etc. Not sure where. In a separate readme file somewhere? There can be readme files per folder.